libraries/dagster-teradata/README.md

```diff
@@ -0,0 +1,138 @@
+# dagster-teradata
+
+A dagster module that provides integration with [Teradata Vantage](https://www.teradata.com/).
+
+## Installation
+The `dagster_teradata` module is available as a PyPI package - install with your preferred python 
+environment manager.
+
+```
+source .venv/bin/activate
+pip install dagster-teradata
+```
+
+## Example Usage
+
+This offers seamless integration with Teradata Vantage, facilitating efficient workflows for data processing, management, 
+and transformation. This module supports a range of scenarios, such as executing queries, managing tables, 
+and integrating with cloud storage solutions like AWS S3 and Azure Data Lake Storage (ADLS). Additionally, 
+it enables compute cluster management for Teradata Vantage Cloud Lake.
+
+```python
+import os
+import pytest
+from dagster import job, op
+from dagster_teradata import TeradataResource
+
+td_resource = TeradataResource(
+    host=os.getenv("TERADATA_HOST"),
+    user=os.getenv("TERADATA_USER"),
+    password=os.getenv("TERADATA_PASSWORD"),
```

Comment:
The [recommended](https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets#using-envvar-vs-osgetenv) approach is to use `EnvVar` instead